src/redemptions/approve-list.ts

```diff
@@ -0,0 +1,94 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import {
+  DataChecksLabel,
+  ExtractionLabel,
+  ImageChecksLabel,
+  JumioTransactionRetrieveResponse,
+  LivenessLabel,
+  SimilarityLabel,
+  UsabilityLabel,
+  WatchlistScreeningLabels,
+} from '../jumio-api/interfaces/jumio-transaction-retrieve';
+
+// ADD APPROVED LABEL SETS HERE AND THEY WILL BE TREATED AS SUCCESS
+// IF A TYPE LABEL IS NOT INCLUDED, IT IS ASSUMED THE VALUES MUST BE 'OK'
+export const BENIGN_FAILURES: ApprovedLabelSet[] = [
+  {
+    maxRiskScore: 50,
+    usabilityLabels: ['NOT_UPLOADED', 'OK'],
+    livenessLabels: ['BAD_QUALITY', 'OK'],
+  },
+];
+
+export type ApprovedLabelSet = {
+  maxRiskScore: number;
+  similarityLabels?: SimilarityLabel[];
+  dataChecksLabels?: DataChecksLabel[];
+  extractionLabels?: ExtractionLabel[];
+  usabilityLabels?: UsabilityLabel[];
+  imageChecksLabels?: ImageChecksLabel[];
+  watchlistScreeningLabels?: WatchlistScreeningLabels[];
+  livenessLabels?: LivenessLabel[];
+};
+
+export function matchApprovedLabels(
+  status: JumioTransactionRetrieveResponse,
+): string | null {
+  const approvals = BENIGN_FAILURES.map((approvedLabelSet) =>
+    matchApproveLabelSet(status, approvedLabelSet),
+  ).filter((a) => a);
```

Comment:
This type is wrong, without this it is `(ApprovedLabelSet | null)[]`
```suggestion
  ).filter<ApprovedLabelSet>((a) => a);
```